autogen/agentchat/contrib/web_surfer.py

```diff
@@ -85,6 +106,18 @@ def __init__(
         self.register_reply([Agent, None], ConversableAgent.generate_function_call_reply)
         self.register_reply([Agent, None], ConversableAgent.check_termination_and_human_reply)
 
+    @property
+    def text_content(self):
+        return self.browser.page_content
+
+    @property
+    def render_text(self):
+        self.browser._set_page_content(self.browser.page_content)
```

Comment:
can we avoid using private methods here?

Reply:
I'll find a way to do it, updating coming Friday